### PR TITLE
Migrate additional JPA Query fat test cases.

### PR DESCRIPTION
**Pull request by jgrassel**
Migrate additional Query Fats to Open Liberty.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

File: dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/svlquery/ddl/WEB-INF/classes/JPA_SVLQUERY_POPULATE_DB2.ddl [hunk 1]
```diff
@@ -45,7 +45,7 @@ INSERT INTO ${schemaname}.JPASUPPLIERPARTTAB_JPAPARTTAB (SUPPLIERS_SID,SUPPLIES_
 INSERT INTO ${schemaname}.JPASUPPLIERPARTTAB_JPAPARTTAB (SUPPLIERS_SID,SUPPLIES_PARTNO) VALUES (1,10);
 INSERT INTO ${schemaname}.JPASUPPLIERPARTTAB_JPAPARTTAB (SUPPLIERS_SID,SUPPLIES_PARTNO) VALUES (1,11);
 INSERT INTO ${schemaname}.JPASUPPLIERPARTTAB_JPAPARTTAB (SUPPLIERS_SID,SUPPLIES_PARTNO) VALUES (1,12);
-INSERT INTO ${schemaname}.JPATASKBEAN (TASKID,COST,DESCRIPTION,NAME,PROJECT_PROJID) VALUES (1010,123.45,'Design ','Design ',1000);
+INSERT INTO ${schemaname}.JPATASKBEAN (TASKID,COST,DESCRIPTION,NAME,PROJECT_PROJID) VALUES (1010,123.45,'Design','Design',1000);
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/svlquery/ddl/WEB-INF/classes/JPA_SVLQUERY_POPULATE_DB2.ddl:48]: I think this change should be also made in the HSQL/INFORMIX/MYSQL/SYBASE DDL files for SQLQuery

**Review by dazey3 — CHANGES_REQUESTED** [review]
Just make sure all the database files get the same changes and looks good to me! Thanks!

**Comment by jgrassel** [comment]
#runlibbybot

**Comment by jgrassel** [comment]
#run-libby-bot

**Review by dazey3 — APPROVED** [review]
LGTM!

**Pull request merged**